ISSUE_TEMPLATE/doc_issue.md

```diff
@@ -0,0 +1,34 @@
+### Does the doc defect  apply to a code bug? 
+- Yes __
+  - Specify the issue for the code bug ____.
+  - Add the Doc bug label.
+- No __
+
+### Does the documentation apply to a particular release?
+- Yes ____
+  - Specify the release:____
+- No  ____
+
+### Does the documentation apply to all operating systems? 
+- Yes __  
+- No  __ 
+  - Specify operating systems: ______
+
+### Specify what you want documented (required):
+Complete the following information:
+
+#### To create a topic:
+- Specify a link to the section in the navigation where the topic should go:
+- Specify a first draft of the topic that you want added:
+#### To update existing topics:
+- Specify links to the topics that are affected:
+- Include a copy of the current text and the exact text to which it will change. For example: Change ABC to XYZ:
+
+### Author approvals:
+- If you are verifying steps in a task, complete the following items:
```

Comment:
I don't think the open liberty docs have tasks so I don't think this is needed.

Reply:
We have some issues that are labeled as tasks https://github.com/OpenLiberty/docs/issues?q=is%3Aopen+is%3Aissue+label%3A%22common+task%22   We could end up with a few tasks outside of guides. So, I think it would be good to keep this option in the template. If the issue doesn't deal with a task, the author can skip over this text. Thoughts?

Reply:
I have 2 issues. The first is that I am not convinced we should have tasks the way KC has tasks. However the bigger issue here is that the entire template talks about topics, doesn't talk about the difference between a topic and a task and then all of a sudden the developer is told if you are doing something with a task without defining what that means. To be honest if this is for a technical accuracy review I don't think it is necessary to have different rules for tasks vs other things. Surely the question is about whether the information is technically correct and they shouldn't need to be told different steps in the case that it is a list of steps to follow vs reference vs concept.

Reply:
Well, topics with steps seem to be creeping into the OpenLiberty documentation. So if any topics have steps, they need to be tested..... Good point about tasks versus topics. I found that I had one reference to tasks and so I changed the wording so that I refer to topics, not tasks..... I do need specific rules for steps in a topic. For years, we've had the rule that topics with steps must be tested. Reviewing the steps doesn't suffice. Errors, unclear instructions, and missing missing explanations are caught when the documentation is tested.